dispatcher/DAQController.py

```diff
@@ -284,20 +286,20 @@ def control_detector(self, command, detector, force=False):
                 hosts = (readers, cc) # we want the cc to delay by 1s
                 # we can safely short the logic here and buy an extra logic cycle
                 self.one_detector_arming = False
-                delay = 1.5
+                delay = self.start_cmd_delay
             else: # stop
                 readers, cc = self.mongo.get_configured_nodes(detector,
                     self.goal_state['tpc']['link_mv'], self.goal_state['tpc']['link_nv'])
                 hosts = (cc, readers)
                 if force or self.latest_status[detector]['status'] not in [DAQ_STATUS.RUNNING]:
                     delay = 0
```

Comment:
Don't know if it makes sense also to add a arm delay or a stop delay to the config?

Reply:
We could, but we haven't found a reason to stagger ARMs. We only stagger STOPs if the readout is live so the end of the datastream doesn't get corrupted, otherwise everything gets it at the same time.